internal/vds/vds.go

```diff
@@ -99,6 +99,14 @@ type FenceMetadata struct {
 	Shape []int `json:"shape" swaggertype:"array,integer" example:"10,50"`
 } // @name FenceMetadata
 
+type BadRequestError struct {
```

Comment:
BadRequest is very HTTP ish and I don't like soaking our code with http stuff. What about InvalidArgument?